index.js

```diff
@@ -6,12 +6,39 @@ module.exports = {
   extends: ['airbnb'],
   plugins: ['chai-friendly'],
   rules: {
-    'no-unused-expressions': 0,
-    'chai-friendly/no-unused-expressions': 2,
+    'no-unused-expressions': 'off',
+    'chai-friendly/no-unused-expressions': 'error',
     'linebreak-style': ['error', 'unix'],
     'no-param-reassign': ['error', { props: false }],
     'func-style': ['error', 'declaration', { allowArrowFunctions: true }],
     'no-use-before-define': ['error', { functions: false }],
-    'consistent-return': 0,
+    'no-unused-vars': ['error', {'vars': 'all', 'args': 'none'}],
+    'consistent-return': 'off',
+    'arrow-body-style': ['warn', 'as-needed', {
+      requireReturnForObjectLiteral: false,
+    }],
+    'prefer-rest-params': 'warn',
+    'prefer-destructuring': ['warn', {
+      VariableDeclarator: {
+        array: false,
+        object: true,
+      },
+      AssignmentExpression: {
+        array: true,
+        object: true,
+      },
+    }, {
+      enforceForRenamedProperties: false,
+    }],
+
+    'no-underscore-dangle': 'off',
+    'array-element-newline': 'off',
+    'function-paren-newline': ['warn', 'multiline'],
```

Comment:
I think I'd rather turn this off. Mainly for cases like:
```javascript
function myFunc(param1, param2, param3, param4, param5,
    param6, param7, param8, param9, param10) {
  // do something
}
```
This just looks weird...
```javascript
function myFunc(
  param1,
  param2,
  param3,
  param4,
  param5,
  param6,
  param7,
  param8,
  param9,
  param10
) {
  // do something
}
```

Reply:
im going to sound like one of those people but if you are doing a function with 10 params somethign is wrong with the code

Reply:
True, but this rule won't solve that problem either way.

Reply:
id more see it as an indicator of where the code should be re thought then use the multiline but alright

Reply:
And that's why it was set as a "warn" rather than an "error"